templates/update_conf.j2.py

```diff
@@ -0,0 +1,74 @@
+#!{{ ansible_python_interpreter | default('/usr/bin/python') }}
```

Comment:
is this file still needed?

Reply:
Well it allows you to run it outside of the services, for example if you want to change the maximum number of CPUs allowed by the partition. I think that could be useful

Reply:
it's not copied to the server, though, or am I overlooking something?

Reply:
oh yes sorry, I was thinking this was the other template. I'll delete it